usr/share/rear/prep/CDM/default/450_check_cdm_client.sh

```diff
@@ -0,0 +1,13 @@
+# 450_check_cdm_client.sh
+# 
+# This script checks if a Rubrik CDM client is installed and running
+#
+
+Log "Backup method is Rubrik (CDM): check backup_agent_main"
+if [ ! -x /usr/bin/rubrik/backup_agent_main ]; then
+    Error "Please install Rubrik (CDM) client software."
+fi
+
+ps ax | grep backup_agent_main | grep -v grep  1>/dev/null
+StopIfError $? "Rubrik (CDM) RBS backup_agent_main was not running on this client."
```

Comment:
@DamaniN 
from plain looking at the code I wonder if `StopIfError` works as you indent here
because the exit code of a pipe is the exit code of its last command
so that `StopIfError` tests here the exit code of `grep -v grep`.

Furthermore `1>/dev/null` (or `2>/dev/null`) is usually not needed
because both stdout and stderr are redirected into ReaR's log file
cf. "What to do with stdin, stdout, and stderr" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
I had copied that bit of code from usr/share/rear/prep/NSR/default/450_check_nsr_client.sh. I've retested it and the code works fine. It seems that if any command in the pipe chain fails, the file return code is 1. I moved the group for our agent to the end just to be sure though. I also removed the redirect.